deploy-as-code/helm/environments/qa.yaml

```diff
@@ -528,6 +528,7 @@ property-services:
   egov-idgen-assm-format: "PB-AS-[cy:yyyy-MM-dd]-[SEQ_EG_PT_ASSM]"
   property-userevents-pay-link: "digit-ui/citizen/payment/my-bills/$businessService/$propertyId?tenantId=$tenantId"
   property-decryption-abac-enabled: "true"
+  egov-state-level-tenant-id: "pb"
```

Comment:
@hinamakhija-eGov and @kavi-egov we already have state-level tenant ID in qa.yaml https://github.com/egovernments/DIGIT-DevOps/blob/04c09f9647405e78250c029545ef226db852d795/deploy-as-code/helm/environments/qa.yaml#L39

Why do we need to add again at the service level?  Please recheck.

Reply:
uat property-services is having some issues related to tenantid, need to test the scenario on qa for the same.

Reply:
@hinamakhija-eGov why cannot we use the existing state-level config already defined in qa.yaml? @kavi-egov please check and guide on this.